skiplist/src/arena.rs

```diff
@@ -49,7 +49,7 @@ impl Arena {
         let offset = self.len.fetch_add(size, Ordering::SeqCst);
 
         // Grow the arena if there is no enough space
-        if offset + size > self.cap.get() {
+        if offset > self.cap.get() {
```

Comment:
fetch_add means add to the current value and return the previous value. So offset + size is needed.

Reply:
I have fixed back. Thanks.